src/core/Value.h

```diff
@@ -461,3 +464,36 @@ std::ostream& operator<<(std::ostream& stream, const Value::ObjectType& u);
 using VectorType = Value::VectorType;
 using EmbeddedVectorType = Value::EmbeddedVectorType;
 using ObjectType = Value::ObjectType;
+
+class VirtualValueIterator
+{
+public:
+  // These need to know how the particular iterator works, and so must be implemented in
+  // a subclass.
+  virtual void inc();
```

Comment:
Mark these with `= 0` as they do not have an implementation yet. Same for the virtual methods in `Values`.

Reply:
And why `inc` instead of the typical `++`?

Reply:
I couldn't get ++ to work.  I never figured out why.  Quite possibly prefix ++ versus suffix ++; I found later that on a different instance.

Reply:
I think that was probably it, changed over and it's building OK now.